packages/fiori/test/pages/IllustratedMessage.html

```diff
@@ -70,9 +70,19 @@
 		<ui5-option>SuccessHighFive</ui5-option>
 		<ui5-option>SuccessScreen</ui5-option>
 		<ui5-option>Tent</ui5-option>
-		<ui5-option>UploadCollection</ui5-option>
```

Comment:
is this entry deleted by intention?

Reply:
It was removed unintentionally. Thanks for noticing !